config/locales/client.en.yml

```diff
@@ -600,19 +597,19 @@ en:
     cancel_invitation: Cancel invitation
 
   notification_models:
-    discussion: 'thread: '
-    comment: 'comment in '
-    poll: 'poll: '
-    outcome: 'outcome: '
-    stance: 'vote: '
+    discussion: 'Thread: '
```

Comment:
These are composed into a larger string, again, I would not do it this way if I did it now. but this is old stuff. This change will make it look quite odd.